### ISSUE TITLE
Redesign Describe One Variable

### ISSUE THREAD

**Issue by mmumbo** [IDEMSInternational/R-Instat#2894]
Hi team,
I was trying to create some summaries on measure of central tendency 'Describe one variable' dialogue but i find it a bit unclear on these areas;
(1.) Removing a variable from the receiver dialogue, besides double clicking is it possible to add a left                        arrow as indicated in the screenshot below to remove variable from the receiver dialogue?

(2) Can 'customise' radio button be removed and replaced by something like 'Additional summaries'? it can be a bit confusing by first selecting 'customise' then 'summaries'
![image](https://cloud.githubusercontent.com/assets/11607243/24659576/bc9254be-1955-11e7-9bfd-35976829b341.png)

**Comment by lilyclements** [comment]
I agree with (2). I'm not keen on there being both a "Customise" checkbox and a "Summaries..." button. While I am aware the checkbox means that the base function changes, as a user I may not be aware of this and hence all the checkbox seemingly does is enable the button.

* We can't have just the button as otherwise the user can't change the base function back to summary() without resetting the dialog
* Having just the check box means the user can't go back into the sub unless they check and uncheck the button. (Although I prefer this to the current system)

I'm not sure what to suggest around this problem. Does anyone have any ideas?

**Comment by dannyparsons** [comment]
Thanks for the issue Mike. I agree the dialog is confusing in the current state. Not sure how to make it less confusing though. Possibly radio buttons for a choice of "auto" summaries or "customised"?

On the back arrow point. It was something we discussed initially and thought we preferred to keep the dialogs simple and not have it if we don't need it. There is double clicking as well as backspace and delete keys to delete. 
It will be a good thing to ask users though to see if it is too confusing without the back button. If people feel strongly we can consider putting it in.

Keep the comments and suggestions coming, it's useful to have this feedback.

**Comment by lilyclements** [comment]
Closing since we have moved past this design
